managed/utils/envvars/parser.go

```diff
@@ -173,7 +173,7 @@ func ParseEnvVars(envs []string) (*models.ChangeSettingsParams, []error, []strin
 		case "PMM_VM_URL":
 			_, err = url.Parse(v)
 			if err != nil {
-				err = fmt.Errorf("invalid value %q for environment variable %q", v, k)
+				errs = append(errs, fmt.Errorf("invalid value %q for environment variable %q", v, k))
```

Comment:
@BupycHuk 
The `err` was not being used, hence the fix. Should we add `continue` here as well?

Reply:
Continue isn't required, because we don't have any action after